spec/defines/system_user_spec.rb

```diff
@@ -11,14 +11,19 @@
 		:operatingsystemmajrelease => '8',
                 :os                        => { :distro => { :codename => 'wheezy' }, :family => 'Debian', :name => 'Debian', :release => { :major => '7', :full => '7.0' } }
 	} }
-
+  
   context 'with default' do
     let(:params) { {'create_user' => true} }
     it { should contain_user('testuser') }
     it { should contain_exec('docker-system-user-testuser').with_command(/docker testuser/) }
     it { should contain_exec('docker-system-user-testuser').with_unless(/grep -qw testuser/) }
   end
 
+  context 'when adding a system user' do
```

Comment:
@MWilsonPuppet we don't need this test. The one you've added in the docker spec is sufficient. This one is failing because you're passing in a param that doesn't actually exist in the define type, not does the exec. A better place to test this would be in the service_spec.rb